### DISCUSSION TITLE
Unit 6.2 video 2 - How to interpret loss vs learning rate graph?

### DISCUSSION THREAD

**Discussion by XinjieLu** [Lightning-AI/dl-fundamentals#115] [Lecture videos and quizzes]
Hi,
I'm confused about how to interpret the graph. Why shall we choose the point with the steepest downward slope instead of the point with the lowest loss? Thanks.
@rasbt 

![unit6 2-video2](https://github.com/user-attachments/assets/d5a15662-01ae-416a-b5f1-ddf50a139d49)

**Comment by rasbt** [comment]
That's a good question. Usually, the point at the lowest here has a learning rate that is too high so even if the loss is initially going down faster it can lead to unstable training behavior.
